src/test/chainstate_write_tests.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024- The Bitcoin Core developers
```

Comment:
I think the convention is:
```suggestion
// Copyright (c) 2024-present The Bitcoin Core developers
```

Reply:
Removed the dates altogether.